README.md

```diff
@@ -2,6 +2,8 @@
 
 Generate JUnit compatible XML reports in Rust.
 
+NEW! Now with output format compatible with GitLab and Jenkins.
+
```

Comment:
Please put this in the [changelog in the unrelease section](https://github.com/bachp/junit-report-rs/blob/master/CHANGELOG.md#unreleased) instead of the readme.

Reply:
Done!
Thanks for the quick response!